### PR TITLE
Completed TL translations for client-svn.asc - 1,217 words

### PR DESCRIPTION
**Pull request by josteinfrancois**
This is the 3nd and final part of my translation in client-svn.asc.

### CODE REVIEW

File: book/09-git-and-other-scms/sections/client-svn.asc [hunk 1]
```diff
@@ -354,39 +354,39 @@ Successfully followed parent
...
-It's important to note that it doesn't check you out into that branch; if you commit at this point, that commit will go to `trunk` on the server, not `opera`.
+Ginagawa nito ang katumbas ng `svn copy trunk branches/opera` na utos sa Subversion at tumatakbo sa Subversion na server.
+Importanteng tandaan na ito ay hindi nagchi-check out sa iyo sa branch na iyon; kung ikaw ay nag-commit sa puntong ito, ang commit na iyon ay pupunta sa `trunk` sa server, hindi sa `opera`.
 
-===== Switching Active Branches
+===== Pagpapalit sa Aktibong mga Branch
 
-Git figures out what branch your dcommits go to by looking for the tip of any of your Subversion branches in your history – you should have only one, and it should be the last one with a `git-svn-id` in your current branch history.
+Inaalam ng Git kung anong branch pupunta ang iyong mga dcommit sa pamamagitan ng paghahanap ng dulo ng anuman sa iyong Subversion na mga branch sa iyong kasaysayan – dapat kang magkaroon ng isa lamang, at ito dapat ang huli na may isang `git-svn-id` sa iyong kasalukuyang kasaysayan ng branch.
```
Comment (robin-ho) [inline book/09-git-and-other-scms/sections/client-svn.asc]: dcommit to commit

File: book/09-git-and-other-scms/sections/client-svn.asc [hunk 2]
```diff
@@ -456,35 +456,35 @@ Last Changed Rev: 87
...
+Sa paraang iyon, hindi mo kakalatan ang proyekto ng `.gitignore` na mga file.
+Ito ay isang magandang opsyon kung ikaw lamang ang gumagamit ng Git sa isang Subversion na koponan, at iyong mga kasama sa koponan ay hindi gustong gumamit ng `.gitignore` na mga file sa proyekto.
 
-===== Git-Svn Summary
+===== Git-Svn na Buod
 
-The `git svn` tools are useful if you're stuck with a Subversion server, or are otherwise in a development environment that necessitates running a Subversion server.
-You should consider it crippled Git, however, or you'll hit issues in translation that may confuse you and your collaborators.
-To stay out of trouble, try to follow these guidelines:
+Ang `git svn` na kasangkapan ay kapaki-pakinabang kung ikaw ay nananatili sa isang Subversion na server, o kaya ay nasa isang development na environment na nangangailangan ng pagpapatakbo ng isang Subversion na server.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/client-svn.asc]: na kasangkapan to na mga kasangkapan

**Comment by mind-blast** [comment]
so far only seen one suggestion.

File: book/09-git-and-other-scms/sections/client-svn.asc (hunk 2 above)
Comment (josteinfrancois) [reply book/09-git-and-other-scms/sections/client-svn.asc]: updating it now

**Comment by josteinfrancois** [comment]
Hey @robin-ho . Already updated based on your suggestion. Thank you.

**Pull request merged**